webpack.js

```diff
@@ -0,0 +1,14 @@
+const { merge } = require('webpack-merge')
+const path = require('path')
+const webpackConfig = require('@nextcloud/webpack-vue-config')
+
+const config = {
+	entry: {
+		settings: path.resolve(path.join('src', 'settings.js'))
```

Comment:
Will most likely throw an error stating the 'main' entry file doesn't exists?

Reply:
Just have a look 5 lines below... 😉 
```
delete mergedConfig.entry.main
```

Reply:
Next step will then be to have a second entry `settings-sharing`, which is why i chose this way instead of using the main-entry.